src/common/Header/styled.js

```diff
@@ -0,0 +1,154 @@
+import styled from "styled-components";
+import { ReactComponent as VideoSVG } from "../Header/Icons/video.svg";
+import { ReactComponent as SearchSVG } from "../Header/Icons/search.svg";
+
+export const Wrapper = styled.header`
+  max-width: 1920px;
+  margin: 0 auto;
+  background-color: #18181b;
+  color: white;
+  margin-bottom: 58px;
+`;
+
+export const Content = styled.div`
+  max-width: 1260px;
+  margin: 0 auto;
+  padding: 23px;
+  display: flex;
+  justify-content: space-between;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    padding: 16px;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    flex-direction: column;
+  }
+`;
+
+export const Container = styled.div`
+  display: grid;
+  grid-template-columns: auto auto;
+  grid-gap: 80px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.medium}) {
+    grid-gap: 0;
+  }
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.large}) {
+    justify-content: space-between;
+    margin-bottom: 20px;
+  }
+`;
+
+export const Title = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const Link = styled.a`
+  display: flex;
+  align-items: center;
+  font-size: 24px;
+  font-weight: 500;
+  cursor: pointer;
+  letter-spacing: -1.5px;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
```

Comment:
letter-spacing for mobile should be  -0.5px

Reply:
Ok. I will correct it. Thanks 😄